docs/index.html

```diff
@@ -3,9 +3,13 @@
 <head>
   <meta charset="utf-8" />
   <title>PokéSprite</title>
+  <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"/>
   <link type="text/css" href="build/pokesprite.min.css" rel="stylesheet" media="screen" />
-  <script charset="utf-8" src="build/pokesprite.min.js" ></script>
   <link type="text/css" href="build/overview.min.css" rel="stylesheet" media="screen" />
+  <style>

```

Comment:
Since this is an inline style tag with some extra styles, and not a minified stylesheet, let's use standard indentation. Also, it should have a MIME type like the others.